docs/src/main/paradox/running.md

```diff
@@ -0,0 +1,63 @@
+# Running a Projection
+
+Once you have decided how you want to build your projection, the next step is to run it. Typically, you run it in a distributed fashion in order to spread the load over the different nodes in an Akka Cluster. However, it's also possible to run it as a single instance (when not clustered) or as single instance in a Cluster Singleton.
+
+## Dependencies
+
+To distribute the projection over the cluster we recommend the use of [SharderDaemonProcess](https://doc.akka.io/docs/akka/current/typed/cluster-sharded-daemon-process.html). Add the following dependency in your project if not yet using Akka Cluster Sharding:
+
+@@dependency [sbt,Maven,Gradle] {
+  group=com.typesafe.akka
+  artifact=akka-cluster-sharding-typed_$scala.binary.version$
+  version=2.6.5
```

Comment:
`$akka.version$`

Reply:
yeay, I just copy and pasted from the browser. If I had looked the source, I would have learn something.